### PR TITLE
Wazuh Kibana plugin troubleshooting update

### PR DESCRIPTION
**Pull request by s-ocando**
## Description

This PR updates the Wazuh Kibana plugin troubleshooting section. 

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [ ] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

## Note to the reviewer

This PR includes changes to the `redirect.js` script that need to be included in all production branches.

### CODE REVIEW

File: source/user-manual/kibana-app/troubleshooting.rst [hunk 1]
```diff
@@ -29,147 +48,168 @@ Elasticsearch needs a specific template to store Wazuh alerts, otherwise visuali
...
 
   {"acknowledged":true}
 
-.. warning::
-  Indices with the wrong template will need to be reindexed. You can follow our :ref:`reindexation guide <restore_alerts_2.x_3.x>`.
+If this error occurs after an **upgrade from a 3.x version** the solution is to remove the ``wazuh-alerts-3.x-*`` index pattern. Since Wazuh 4.x, the index pattern is ``wazuh-alerts-*``, and you need to remove the old pattern for the new one to take its place.
+
+.. code-block:: console
+   
+   # curl 'https://<kibana_ip>:<kibana_port>/api/saved_objects/index-pattern/wazuh-alerts-3.x-*' -X DELETE  -H 'Content-Type: application/json' -H 'kbn-version: 7.10.2' -k -uelastic:<elastic_password>
```
Comment (frankeros) [inline source/user-manual/kibana-app/troubleshooting.rst:62]: `-uelastic:<elastic_password>` should be `-u elastic:<elastic_password>`

File: source/user-manual/kibana-app/troubleshooting.rst [hunk 2]
```diff
@@ -29,147 +48,168 @@ Elasticsearch needs a specific template to store Wazuh alerts, otherwise visuali
 
 .. code-block:: console
 
-  # curl https://raw.githubusercontent.com/wazuh/wazuh/v|WAZUH_LATEST|/extensions/elasticsearch/7.x/wazuh-template.json | curl -X PUT "http://localhost:9200/_template/wazuh" -H 'Content-Type: application/json' -d @-
+  # curl https://raw.githubusercontent.com/wazuh/wazuh/v|WAZUH_LATEST|/extensions/elasticsearch/7.x/wazuh-template.json | curl -X PUT "https://localhost:9200/_template/wazuh" -H 'Content-Type: application/json' -d @- -u <user>:<password> -k
```
Comment (frankeros) [inline source/user-manual/kibana-app/troubleshooting.rst:51]: what `user` is it referring to?

File: source/user-manual/kibana-app/troubleshooting.rst [hunk 3]
```diff
@@ -29,147 +48,168 @@ Elasticsearch needs a specific template to store Wazuh alerts, otherwise visuali
...
 
-API version mismatch. Expected vX.Y.Z
--------------------------------------
+The first step is to check if there are alerts in Elasticsearch.
 
-The Wazuh Kibana plugin uses the Wazuh API to fetch information, being compatible between patch versions. For example, you can use an Wazuh Kibana plugin designed for Wazuh 3.7.2 with a Wazuh API 3.7.1.
+.. code-block:: console
 
-You can't use the 3.7.2 version of Wazuh API with a Wazuh app designed for Wazuh |WAZUH_LATEST|.
+  # curl https://<ELASTICSEARCH_IP>:9200/_cat/indices/wazuh-alerts-* -u <username>:<password> -k
```
Comment (frankeros) [inline source/user-manual/kibana-app/troubleshooting.rst:117]: same, which `username`?

**Review by frankeros — COMMENTED** [review]
it looks good, I left some comments

File: source/user-manual/kibana-app/troubleshooting.rst (hunk 2 above)
Comment (s-ocando) [reply source/user-manual/kibana-app/troubleshooting.rst]: Replaced by <elasticsearch_user>:<elasticsearch_password>

File: source/user-manual/kibana-app/troubleshooting.rst (hunk 3 above)
Comment (s-ocando) [reply source/user-manual/kibana-app/troubleshooting.rst]: Replaced by <elasticsearch_user>:<elasticsearch_password>

**Review by frankeros — APPROVED** [review]
LGTM!  GJ! :tada:

File: source/user-manual/kibana-app/troubleshooting.rst [hunk 4]
```diff
@@ -29,147 +48,168 @@ Elasticsearch needs a specific template to store Wazuh alerts, otherwise visuali
...
+If you have a custom index pattern, make sure to replace it accordingly.
+
+**Very important:** Clean the browser’s cache and cookies.
+
 
 Wazuh API seems to be down
 --------------------------
 
-It means your Wazuh API could be unavailable. Since the Wazuh Kibana plugin needs data from the Wazuh API, it must be available for the Wazuh Kibana plugin.
+This issue means that your Wazuh API might be unavailable. Check the status of the Wazuh manager to check is the service is active:
```
Comment (davidfmiranda) [inline source/user-manual/kibana-app/troubleshooting.rst:73]:
```suggestion
This issue means that your Wazuh API might be unavailable. Check the status of the Wazuh manager to check if the service is active: 
```

**Review by davidfmiranda — CHANGES_REQUESTED** [review]
Hello @s-ocando, I've reviewed your PR and LGTM. I've suggested one quick fix.

Regards,

David

**Review by davidfmiranda — APPROVED** [review]
LGTM

**Pull request merged**